gems/sorbet/test/snapshot/partial/require_relative/expected/out.log

```diff
@@ -81,4 +81,4 @@ If instead you want to explore your files locally, here are some things to try:
   Then, run srb tc and try to fix any errors.
 
 Please don't hesitate to give us your feedback!
-    
+
```

Comment:
if you're going to fix this, you need to fix it in the thing that makes it since this is just a recording file

Reply:
Looks like https://github.com/sorbet/sorbet/pull/967/files#diff-6703ba5d1f2495868b98dbbdace7e72fR163

Reply:
Ah, where would I find that? Looks like I need to remind Sublime to stop being so happy with killing whitespace.